### PR TITLE
fix(useractions): clear all local settings on logout

### PR DESCRIPTION
**Pull request by anthony-liddle**
## Goal

Clear all locally stored settings when a user logs out

## Todos:
- [x] Clear all settings on logout
- [x] Sneaking in localization for Log In context menu button

## Implementation Decisions

Easy enough to clear everything instead of just the access token.

### CODE REVIEW

File: src/common/_locales/de/strings.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
   "context_menu" : {
     "open_list" : "Deine Pocket-Liste öffnen",
     "discover_more" : "Entdecke noch mehr interessante Inhalte auf Pocket",
+    "log_in" : "Einloggen",
```
Comment (anthony-liddle) [inline src/common/_locales/de/strings.json:23]: Will be helpful soon...

File: src/pages/background/index.js [hunk 2]
```diff
@@ -78,9 +77,6 @@ chrome.runtime.onMessage.addListener(function (message, sender) {
     case USER_LOG_IN:
       handle.logIn(tab)
       return
-    case USER_LOG_OUT:
-      handle.logOut(tab)
-      return
```
Comment (anthony-liddle) [inline src/pages/background/index.js:83]: This isn't being fired anywhere in app

File: src/pages/background/userActions.js [hunk 3]
```diff
@@ -155,7 +155,7 @@ export function logOut() {
 }
 
 export function loggedOutOfPocket() {
-  removeSettings(['access_token'])
+  chrome.storage.local.clear()
```
Comment (anthony-liddle) [inline src/pages/background/userActions.js:158]: Easy-peasy

Comment (wtfluckey) [reply src/pages/background/userActions.js:158]: Is `removeSettings` being used anywhere else?

Comment (anthony-liddle) [reply src/pages/background/userActions.js:158]: Nope! good call. I deleted it from `interface`

**Pull request merged**